libmscore/element.h

```diff
@@ -159,7 +159,10 @@ class EditData {
 ///     score layout elements.
 //-------------------------------------------------------------------
 
+enum class ElementScope { VOICE, STAFF, PART, SYSTEM };
+
 class Element : public ScoreElement {
+      ElementScope _scope { ElementScope::STAFF };
```

Comment:
Would it make more sense to implement this as a virtual function that returns `ElementScope::STAFF` in `class Element`, but can be overridden in `class Pedal` to return `ElementScope::PART`?

Reply:
It might. I'll have to think a bit. I want this to enable users to set the scope for which the element has effect in the inspector. Hard-coding it in the class definition for each element won't work for that. My other option was to make the `_scope` member protected and only implementing accessors where I want to expose the value publicly as in dynamics, so that the Inspector can set its scope, or haripins and cresc./dim. lines).